controllers/azurecluster_reconciler.go

```diff
@@ -71,58 +65,42 @@ func newAzureClusterReconciler(scope *scope.ClusterScope) *azureClusterReconcile
 
 // Reconcile reconciles all the services in pre determined order
 func (r *azureClusterReconciler) Reconcile(ctx context.Context) error {
-	klog.V(2).Infof("reconciling cluster %s", r.scope.ClusterName())
 	if err := r.createOrUpdateNetworkAPIServerIP(); err != nil {
 		return errors.Wrapf(err, "failed to create or update network API server IP for cluster %s in location %s", r.scope.ClusterName(), r.scope.Location())
 	}
 
 	if err := r.setFailureDomainsForLocation(ctx); err != nil {
-		return errors.Wrapf(err, "failed to get availability zones for cluster %s in location %s", r.scope.ClusterName(), r.scope.Location())
+		return errors.Wrapf(err, "failed to get availability zones")
 	}
 
+	r.scope.SetControlPlaneIngressRules()
```

Comment:
this should probably move to a webhook but because the APIServer port is part of Cluster, not AzureCluster so requires a little more work, will refrain from changing that for now leave it as-is (just moved the below generateControlPlaneIngressRules func to cluster.go)